config.yaml

```diff
@@ -0,0 +1,107 @@
+default:
+  compat_provisioners:
+    - _id: 'compat-v5-0'
+      enabled: true
+      version: 13.1.31
+      type: chef_solo
+      log_level: debug
+      # Please update right cookbooks, role, nodes, data_bags and run_list path in you entry config
+      cookbooks_path:
+        - teracy-dev-v05-compat/vendor-cookbooks
+        - teracy-dev-v05-compat/main-cookbooks
+      roles_path: teracy-dev-v05-compat/roles
+      nodes_path: teracy-dev-v05-compat/nodes
+      data_bags_path: teracy-dev-v05-compat/data_bags
+      run_list:
+      - vim
+      - teracy-dev
+      json:
+        docker:
+          enabled: true
+          version: ''
+          package_options: "--force-yes -o Dpkg::Options::='--force-confold' -o Dpkg::Options::='--force-all'"
+          repo: main
+          members:
+          - vagrant
+          action: create
+        docker_registry:
+          force: false
+          entries: []
+        docker_compose:
+          version: ''
+          enabled: true
+        docker_machine:
+          enabled: false
+          release: v0.8.2
+        teracy-dev:
+          directories:
+          - _id: '0'
+            path: "/home/vagrant/workspace"
+            owner: vagrant
+            group: vagrant
+            mode: '0775'
+            action: create
+          aliases:
+          - _id: '0'
+            name: ws
+            command: cd ~/workspace
+            action: add
+          - _id: '1'
+            name: ctop
+            command: docker run -ti --name ctop --rm -v /var/run/docker.sock:/var/run/docker.sock
+              quay.io/vektorlab/ctop:latest
+            action: add
+          - _id: '2'
+            name: http
+            command: docker container run -it --rm --net=host teracy/httpie-jwt-auth:latest-alpine
+            action: add
+          - _id: '3'
+            name: https
+            command: http --default-scheme=https
+            action: add
+          env_vars:
+          - _id: '0'
+            key: EDITOR
+            value: vim
+            action: add
+          inotify:
+            max_user_watches: 524288
+          proxy:
+            enabled: true
+            certs:
+              enabled: true
+              sources:
+              - nginx/certs/default.crt
+              - nginx/certs/default.key
+              destination: "/etc/nginx/certs"
+              owner: vagrant
+              group: vagrant
+              mode: '0644'
+            container:
+              enabled: false
+              name: nginx-proxy
+              repo: jwilder/nginx-proxy
+              tag: alpine
+              volumes:
+              - "/var/run/docker.sock:/tmp/docker.sock:ro"
+              - "/etc/nginx/certs:/etc/nginx/certs"
+              restart_policy: always
+              port:
+              - 80:80
+              - 443:443
+  vm:
+    networks:
+      - _id: "0"
+        type: "public_network"
+    synced_folders:
+      - id: "compat-v05-0"
```

Comment:
remember to install and enable vagrant-gatling-rsync and vagrant-rsync-back plugins